README.md

```diff
@@ -114,6 +114,40 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+#### Publishing messages with schema
+
+Ably Kafka connector supports messages with schema by converting schematic messages to JSON. Produced JSON is then passed as data to Ably message. JSON is produced by eliminating schema fields from the original message.
```

Comment:
> JSON is produced by eliminating schema fields from the original message.

Can you clarify what you mean by "eliminate"?

Reply:
I meant to convey the message to 'get rid of type and other informations from schema' but when I look, I think it's not clear. I made a change here a4942213ba9bd4e516e3ea8afb110f323453a9f4 to hopefully make it a bit more clear but honestly I couldn't find a way to explain this better. I would be open to any suggestions.